src/command/subgraph/push.rs

```diff
@@ -32,6 +32,11 @@ pub struct Push {
     #[structopt(long)]
     #[serde(skip_serializing)]
     service_name: String,
+
+    /// Url of a running service that a gateway can send operations to (often a deployed service).
+    /// May be left empty ("") or a placeholder url if not running a gateway in managed federation mode
```

Comment:
is there a way we can detect if they are using managed or unmanaged federation? would be good to not make them specify at all for the first time if it's not needed.

Reply:
If they are pushing partial schemas to the registry then it will be managed federation. Unmanaged federation doesn't go through the registry. 
Unmanaged federation directly get the subgraph schemas by querying the urls at runtime and fetching them via introspection, so it entirely bypasses the registry.